lib/auth/MatrixAuthProvider.js

```diff
@@ -118,7 +131,16 @@ class MatrixAuthProvider {
             {
                 type: 'm.room.join_rules',
                 content: { join_rule: joinRule }, // can be set to either public, invite or knock
-            }],
+            },
+            {
+                type: 'm.space.parent',
```

Comment:
I think it's important to now note that: we will always add this.
When calling `createRoom` without a parent, we will still add this `m.space.parent` with an empty `state_key`, right?

I guess this is fine right now. But important to remember when we might want to create rooms that do -not- belong to a parent.